.github/dependabot.yml

```diff
@@ -16,3 +16,10 @@ updates:
       - "rollkit/celestia"
     labels:
       - T:dependencies
+  - package-ecosystem: docker
+    directory: "/docker"
+    schedule:
+      interval: weekly
+    open-pull-requests-limit: 10
+    reviewers:
```

Comment:
I'm not sure the impact of this. Will this overwrite any other review request automation that is set up?

I'm wondering if it is better to remove this and keep all the review request logic in a single source of truth for the repo. 

I also haven't used this feature of dependabot, and I don't think we use it anywhere else currently so it might just add some noise

Reply:
I think it only adds docker-related dependabot checks for files in `/docker` directory.

Reply:
I copy & pasted it from the current setup which was introduced here: https://github.com/rollkit/rollkit/commit/5bf2de6f8be96e8144ff8ecefe23cde1e85c1dbf

I'm not aware of the request automation in github. If there is some automation that adds the rollkit team as a reviewer I'm okay with removing it from the dependabot file.